### PR TITLE
Image attachment

### PR DESCRIPTION
**Pull request by tochman**
pt story: https://www.pivotaltracker.com/story/show/179419280

### CODE REVIEW

File: spec/requests/api/articles/create_spec.rb [hunk 1]
```diff
@@ -10,7 +13,7 @@
     before do
       post '/api/articles',
            params: { article:
-            { title: 'Test Article', body: 'This is a test article' } },
+            { title: 'Test Article', body: 'This is a test article', image: image } },
            headers: valid_auth_headers_for_user
     end
```
Comment (SevaDer14) [inline spec/requests/api/articles/create_spec.rb:19]: can add test for attaching image
```
it 'is expected to have attached an image' do
      article = Article.last
      expect(article.image.attached?).to eq true
    end
```

**Review by SevaDer14 — CHANGES_REQUESTED** [review]
seems good, can add one more test.

File: spec/models/image_spec.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+RSpec.describe Image, type: :model do
+  let(:article) { build(:article) }
+  describe 'Factory' do
+    it 'is expected to have valid basic Factory' do
+      expect(create(:image, article: article)).to be_valid
+    end
+
+    # it 'is expected to have valid basic Factory' do
+    #   expect(build(:associated_image)).to be_valid
+    # end
```
Comment (SevaDer14) [inline spec/models/image_spec.rb:10]: comment

Comment (tochman) [reply spec/models/image_spec.rb]: Done

**Review by SevaDer14 — APPROVED** [review]

**Pull request merged**